scalding-commons/src/main/scala/com/twitter/scalding/commons/source/LzoGenericScheme.scala

```diff
@@ -87,6 +87,22 @@ object LzoGenericScheme {
 
   def apply[M](conv: BinaryConverter[M], clazz: Class[M]): LzoGenericScheme[M] =
     new LzoGenericScheme(conv, clazz)
+
+  /**
+   * From a Binary Converter passed in configure in the JobConf using of that by ElephantBird
+   */
+  def setConverter[M](conv: BinaryConverter[M], conf: JobConf, overrideConf: Boolean = false): Unit = {
+    if ((conf.get(ConfigBinaryConverterProvider.ProviderConfKey) == null) || overrideConf) {
+      val extern = Externalizer(conv)
+      try {
+        ExternalizerSerializer.inj.invert(ExternalizerSerializer.inj(extern)).get
```

Comment:
we should move this to `ExternalizerSerialization.testRoundTrip(i: Any): Try[Unit]`